overrides/macos-override.json

```diff
@@ -718,6 +718,9 @@
         "disableFireAnimation": {
             "state": "enabled"
         },
+        "newFeedbackForm": {
```

Comment:
I don't think we need a features/ file here (as it's a short lived flag that's not going to be shared).

Can you add `"exceptions": []` here and then we should pass the tests to get this shipped as macOS only.

Reply:
I agree - if this is macOS only, let's keep it to macOS, unless you know there's going to be Windows counterpart soon (tho - it can be also implemented in config when needed)

Reply:
Discussed here, we're dropping the "new" https://app.asana.com/1/137249556945/task/1210733970843924?focus=true and setting a task reminder to drop if no longer used as well.